include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3280,7 +3296,9 @@ namespace internal
   template <typename VectorType>
```

Comment:
You could similarly remove the body from `BlockVectorSelector` while you're at it.